app/__init__.py

```diff
@@ -695,16 +695,9 @@ def serve_static(filename):
         import os
         from flask import send_from_directory
         # Use the explicit static folder path since we disabled Flask's static handling
-        
-        # Check if we're in Docker (production) or local development
-        docker_static_dir = '/app/static'
-        local_static_dir = os.path.join(os.path.dirname(__file__), 'static')
-        
-        if os.path.exists(docker_static_dir):
-            static_dir = docker_static_dir
-        else:
-            static_dir = local_static_dir
-            
```

Comment:
This is the actual fix for the static files, my editor trims automatically trailing empty spaces, that's why the other changes